packages/syft/src/syft/core/node/new/serializable.py

```diff
@@ -12,12 +12,12 @@
 module_type = type(syft)
 
 
-T = TypeVar("T")
+T = TypeVar("T", bound=type)
 
 
 def serializable(
-    attrs: Sequence[str] = [],
```

Comment:
good spot, these should be picked up by bugbear, i swapped flake8 for ruff but maybe it doesnt have these checks... hrm